src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--start-day_mon calendar--month-length_31">
```

Comment:
it's better to change the style of writing the class modifiers:
instead of `calendar--start-day_mon` use `calendar--start-day--mon`
`calendar--month-length_31` -> `calendar--month-length--31`